database/structure.sql

```diff
@@ -0,0 +1,128 @@
+CREATE TABLE users(
+  id SERIAL PRIMARY KEY,
+  username VARCHAR(255),
+  password VARCHAR(255),
+  role_id INTEGER NOT NULL
+);
+
+CREATE TABLE roles(
+  id SERIAL PRIMARY KEY,
+  name VARCHAR(255),
+  create_case BOOLEAN,
+  view_case BOOLEAN,
+  edit_case BOOLEAN,
+  close_case BOOLEAN
+);
+
+CREATE TABLE case_workers(
+  id SERIAL PRIMARY KEY,
+  person_id INTEGER NOT NULL,
+  user_id INTEGER NOT NULL
+);
+
+CREATE TABLE citizens(
+  id SERIAL PRIMARY KEY,
+  person_id INTEGER NOT NULL,
+  cpr INTEGER NOT NULL
```

Comment:
Properly better if it was a VARCHAR, otherwise we will have problems with numbers beginning with a zero.